kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/ExamplesBuildTests.kt

```diff
@@ -24,6 +24,11 @@ internal class ExamplesBuildTests {
         // build only
     }
 
+    @ExamplesTest("jvm/copy-variant", ["koverHtmlReportFirst", "koverHtmlReportSecond"])
+    fun CheckerContext.jvmCopyVariant() {
+        // build only
```

Comment:
Why build only? Shouldn't you check that `first` variant covers only `SubprojectUtils` and `ClassFromSecondProject` is absent?